src/pool/srv_layout.h

```diff
@@ -26,7 +26,7 @@
 #include <daos_types.h>
 
 /* Default layout version */
-#define DS_POOL_MD_VERSION 4
+#define DS_POOL_MD_VERSION 5
```

Comment:
just a quick comment (I haven't reviewed this patch, for example haven't looked at it from a layout version compatibility perspective).

In this patch the DAOS_CONT_MD_VERSION will need to take the same value as DAOS_POOL_MD_VERSION due to https://daosio.atlassian.net/browse/DAOS-8918. So they must be changed together for now.

Reply:
I see, let me bump version together now then.